test/fixtures/with-commands/commands/my-custom-command.js

```diff
@@ -0,0 +1,23 @@
+import consola from 'consola'
+
+export default {
+  name: 'my-custom-command',
+  description: 'My Custom Command',
+  usage: 'my-custom-command <foobar>',
```

Comment:
Double emphasize that we should consider command namespacing with command object schema :)

Reply:
@pi0 gotcha, will make this change.